op-supervisor/supervisor/backend/db/fromda/update.go

```diff
@@ -238,7 +238,7 @@ func (db *DB) addLink(derivedFrom eth.BlockRef, derived eth.BlockRef, invalidate
 		return fmt.Errorf("cannot add block (%s derived from %s), last block (%s derived from %s) is too far behind: (%w)",
 			derived, derivedFrom,
```

Comment:
Side Note - I noticed more `derivedFrom` in this file I hadn't noticed previously, but I won't touch it just yet, to keep this PR clean.